app/perftools/management/commands/create_page_cache.py

```diff
@@ -221,8 +221,8 @@ def create_jtbd_fund_cache():
 
     # WalletConnect / ethers.js / TheDefiant
         # Q(pk=275) | Q(pk=13) | Q(pk=567)
-    id_tuple= (275,13,567)
-    # id_tuple= (6,13,4)
+    # id_tuple= (275,13,567)
```

Comment:
^ remove ?

Reply:
yes I left a todo note
> TODO switch grants query ids before deploy
As that works on my local and will need to pick some from staging and change it back to production ones before deploy